python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -58,6 +62,19 @@ class AssetKeyPartitionKey(NamedTuple):
     partition_key: Optional[str] = None
 
 
+# This is currently used only for the asset partition wipe codepath. In the future, we can rename
+# to AssetPartitionRange or similar for more general use.
+class AssetPartitionWipeRange(NamedTuple):
+    """An AssetKey with a partition range."""
+
+    asset_key: AssetKey
+    partition_range: PartitionKeyRange
+
+    @property
+    def is_universal(self) -> bool:
+        return self.partition_range == UNIVERSAL_PARTITION_KEY_RANGE
+
```

Comment:
Also weary of this `UNIVERSAL_PARTITION_KEY_RANGE` without further examples of how it is actually used. 

If it is "universal" but I access `partition_range` directly, is that valid? I'd rather introduce as few as new things as possible here.

Reply:
It was used upstack but I've removed it and now am just using `None` (as with `AssetKeyPartitionKey`) to represent the whole asset. We can make the change to "all assets are partitioned" in a coherent way later.